net/grpc/gateway/examples/helloworld/envoy.yaml

```diff
@@ -27,7 +27,9 @@ static_resources:
                   max_grpc_timeout: 0s
               cors:
                 allow_origin_string_match:
-                - prefix: "*"
```

Comment:
Can you point us to where this `prefix` field is being deprecated? I still see it here in the docs: https://www.envoyproxy.io/docs/envoy/latest/api-v2/type/matcher/string.proto#envoy-api-msg-type-matcher-stringmatcher.

Reply:
https://www.envoyproxy.io/docs/envoy/latest/api-v2/api/v2/route/route_components.proto.html?highlight=allow_origin#route-corspolicy states that allow_origin field has been deprecated in favor of allow_origin_string_match

Reply:
But we are already using `allow_origin_string_match`, right?

Reply:
This is already done back in #716

Reply:
Thanks.  That was in the echo example but the helloworld was unchanged.

Reply:
But the helloworld/envoy.yaml _is_ changed to be using `allow_origin_string_match` already right?

Reply:
My Bad... Generally use `safe_prefix `with `allow_origin_string_match` . Closing.